packages/harmony/src/components/button/FilterButton/FilterButton.tsx

```diff
@@ -162,9 +153,13 @@ export const FilterButton = forwardRef<HTMLButtonElement, FilterButtonProps>(
       }
     }, [selection, variant, setIsOpen, setSelection])
 
-    const handleOptionSelect = useCallback((option: FilterButtonOption) => {
-      setSelection(option)
-    }, [])
+    const handleOptionSelect = useCallback(
+      (option: FilterButtonOption) => {
+        setSelection(option.value)
```

Comment:
In the mobile version below, we removed `setSelection()` because the consumer of this component is supposed to pass the value in all the time. I'm curious why this version is keeping the value stored in internal state?

Reply:
the `useControlled` hook allows this component to still be uncontrolled and maintain local state. I didn't bother updating the mobile version to include `useControlled` because I was lazy but I could!

Reply:
Ah I see. I missed that we were adding the ability for it to be either controlled or uncontrolled. I thought we were switching it to only allow a controlled usage. I do think it would be nice for the behavior to be consistent across platforms if possible!